build.bat

```diff
@@ -0,0 +1,85 @@
+:: This script will build a .NET project in Release mode, sign the generated DLLs with a provided PFX certificate,
+:: package the DLLs into a NuGet package, and sign the NuGet package with the provided PFX certificate.
+:: This script also handles pre-run cleanup (will delete old DLLs and NuGet package files)
+
+:: Requirements:
+:: - NuGet is installed on the machine and is accessible everywhere (added to PATH)
+:: - dotnet is installed on the machine and is accessible everywhere (added to PATH) (might be in C:\Program Files\dotnet)
+:: - SnInstallPfx (https://github.com/honzajscz/SnInstallPfx) is installed on the machine and is accessible everywhere (added to PATH)
+
+@ECHO OFF
+
+:: Parse command line arguments
+SET projectName=%1
+SET certFile=%2
+SET certPass=%3
+
+:: Constants
+SET containerName=EasyPost
+SET buildMode="Release"
+SET buildPlatform="Any CPU"
+
+:: Delete old files
+@ECHO:
+@ECHO Cleaning old files...
+@RD /S /Q lib
+DEL /S /Q /F *.nupkg
+
+:: Install certificate (needed to automate signing later on)
+@ECHO:
+@ECHO (Re-)Installing certificate to system...
+sn -d "%containerName%"
+SnInstallPfx "%certFile%" "%certPass%" "%containerName%" || GOTO :commandFailed
+
+:: Restore dependencies and build solution
+@ECHO:
+@ECHO Restoring and building project...
+dotnet msbuild -property:Configuration="%buildMode%" -property:Platform="%buildPlatform" -target:Rebuild -restore || GOTO :commandFailed
+
+:: Sign the DLLs
+@ECHO:
+@ECHO Signing DLLs with certificate...
+FOR /R lib %%F IN (*.dll) DO (
+    REM ¯\_(ツ)_/¯
```

Comment:
Issue: this comment is worse than the original. It gives no context to what the author meant here. Comments should clearly call out oddities in code where code cannot self document itself. A shrugging emoji does us no good here.